app/js/demo-controller.js

```diff
@@ -92,8 +126,13 @@ export class DemoPaywallController {
               });
         }
       } else {
+        // A list of offers to show to the user, if available.
+        const offers = [];
         // In a simplest case, just launch offers flow.
         this.subscriptions.showOffers();
+        this.subscriptions.getPropensityModule().then(module => {
+          module.sendEvent('offers_shown', {'offers': offers});
```

Comment:
nit: I *believe* you can just do something like this:

module.sendEvent('offers_shown', {offers});

Reply:
since I'm removing offers variable, I will do this module.sendEvent('offers_shown', {'offers' : []}).
I'll add documentation to clarify.